test/sshkit_functional_test.exs

```diff
@@ -97,23 +91,25 @@ defmodule SSHKitFunctionalTest do
       |> SSHKit.path("/var/log")
 
     [{:ok, output, status}] = SSHKit.run(context, "pwd")
-    assert status == 0
     output = stdout(output)
+
+    assert status == 0
     assert output == "/var/log\n"
   end
 
-  @tag skip: true # it produces an error: "sudo not found" on stderr
   @tag boot: 1
   test "user", %{hosts: [host]} do
     adduser(host, "despicable_me")
+    add_user_to_group(host, host.user, "danger")
```

Comment:
The group is set up in the `Dockerfile` in order to allow passwordless `sudo`ing, required for user switching. Hence also the name "danger".

Maybe you've got a different idea though?